test/task.h

```diff
@@ -227,7 +227,6 @@ static void close_walk_cb(uv_handle_t* handle, void* arg) {
 
 UNUSED static void close_loop(uv_loop_t* loop) {
   uv_walk(loop, close_walk_cb, NULL);
-  uv_run(loop, UV_RUN_DEFAULT);
```

Comment:
This is incorrect. The loop is stopped, so the close handles will not be called unless we run it again, so me must do it.